src/ui/layout/mct-tree.vue

```diff
@@ -635,8 +635,7 @@ export default {
         },
         scrollableStyles() {
             return {
-                height: this.availableContainerHeight + 'px',
-                overflow: this.noScroll ? 'hidden' : 'scroll'
+                height: this.availableContainerHeight + 'px'
```

Comment:
I have no idea why, but if I have "My Items" selected and refresh the page, only a subset of the children for "My Items" show up, I swtiched back to master to check and it doesn't do it there. A way to fix this is to add back in overflow, but specify overflow-y:

`'overflow-y': this.noScroll ? 'hidden' : 'scroll'`

Adding that in should handle the issue with the horizontal scroller, but fix this odd issue as well.